src/views/Error/Error.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { i18n } from '../../config'
+
+class Error extends React.Component {
```

Comment:
You can export a function that returns JSX here, since you don't need any of the react lifecycle methods. The `class` is totally fine though